src/app/static/src/tests/components/header/hintrVersionMenu.test.ts

```diff
@@ -49,6 +59,7 @@ describe("Hintr Menu Version", () => {
         expectTranslated(link, "News", "Nouvelles", "Notícias", store);
         expect(link.attributes("href")).toBe("https://naomi.unaids.org/news");
         expect(link.attributes("target")).toBe("_blank");
+        expect(link.attributes("href")).toBe("https://naomi.unaids.org/news");
```

Comment:
Looks like you've added a duplicate line here?

Reply:
Thanks! Removed that now